homeassistant/components/notify/strings.json

```diff
@@ -42,6 +42,28 @@
         }
       }
     },
+    "html5": {
+      "name": "Notifications 'Send a notification with html5'",
+      "description": "Sends a notification message using the html5 service.",
```

Comment:
I'd go w/ HTML5 as display term, hm?

Reply:
Good catch! You're right—'HTML5' as a display term is more consistent with standard capitalization and naming conventions. I'll update this to reflect that.

Reply:
Done.